substratest/factory.py

```diff
@@ -413,6 +413,7 @@ def add_testtuple(self, objective, traintuple_spec, tag=None):
 @dataclasses.dataclass
 class ComputePlanSpec(_BaseComputePlanSpec):
     tag: str
+    cleanModels: bool
```

Comment:
```suggestion
    clean_models: bool
```
Cf my comment in the backend repository, substra backend should accept snake case fields to be consistent with the other endpoints.